src/components/POIForm.component.js

```diff
@@ -187,4 +170,46 @@ class POIForm extends Component {
   }
 }
 
+function FieldGroup({
+  controlId, //indentifier
+  label,
+  inputType, //text vs. textarea vs. file
+  helperText,
+  thisValue,
+  thisChange
+}) {
+  var fieldGroupModule
```

Comment:
Use `let` instead of `var`. `var` is more or less outdated since it does funky stuff with scope. Look [here](https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var-to-declare-a-variable) for more info